python/cudf/cudf/_lib/datetime.pyx

```diff
@@ -62,82 +62,63 @@ def extract_datetime_component(Column col, object field):
     return result
 
 
-def ceil_datetime(Column col, object field):
+cdef libcudf_datetime.rounding_frequency _get_rounding_frequency(object freq):
+    cdef libcudf_datetime.rounding_frequency freq_val
+
+    # https://pandas.pydata.org/pandas-docs/version/0.25.0/reference/api/pandas.Timedelta.resolution.html
```

Comment:
might as well update this

https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.Timedelta.resolution_string.html

Reply:
Thanks for this! I was looking at [`pandas.Timedelta.resolution`](https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.Timedelta.resolution.html) which does not have any docs.